src/include/platform/ConfigurationManager.h

```diff
@@ -90,6 +90,8 @@ class ConfigurationManager
     virtual CHIP_ERROR GetVendorId(uint16_t & vendorId)                                             = 0;
     virtual CHIP_ERROR GetProductName(char * buf, size_t bufSize)                                   = 0;
     virtual CHIP_ERROR GetProductId(uint16_t & productId)                                           = 0;
+    virtual CHIP_ERROR StoreVendorId(uint16_t vendorId)                                             = 0;
+    virtual CHIP_ERROR StoreProductId(uint16_t productId)                                           = 0;
```

Comment:
Storage does NOT belong in the ConfigurationManager API. We have been removing all setters for "fundamental" (i.e. don't change at runtime) values. If this is added, it should only be to the Linux version altogether, not the general API